packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -52,35 +52,42 @@ export default async args => {
     const result = [];
 
     if (platform === "web") {
-        result.push({
-            input: widgetEntry,
-            output: {
-                format: "amd",
-                file: join(outDir, `${outWidgetFile}.js`),
-                sourcemap: !production ? "inline" : false
-            },
-            external: webExternal,
-            plugins: [
-                ...getClientComponentPlugins(),
-                url({ include: imagesAndFonts, limit: 100000 }),
-                sass({ output: production, insert: !production, include: /\.(css|sass|scss)$/, processor }),
-                alias({
-                    entries: {
-                        "react-hot-loader/root": join(__dirname, "hot")
-                    }
-                }),
-                ...getCommonPlugins({
-                    sourceMaps: !production,
-                    extensions: webExtensions,
-                    transpile: production,
-                    babelConfig: {
-                        presets: [["@babel/preset-env", { targets: { safari: "12" } }]],
-                        allowAllFormats: true
-                    },
-                    external: webExternal
-                })
-            ],
-            onwarn
+        ["amd", "es"].forEach(outputFormat => {
+            result.push({
+                input: widgetEntry,
+                output: {
+                    format: outputFormat,
+                    file: join(outDir, `${outWidgetFile}.${outputFormat === "es" ? "mjs" : "js"}`),
+                    sourcemap: !production ? "inline" : false
+                },
+                external: webExternal,
+                plugins: [
+                    ...getClientComponentPlugins(),
+                    url({ include: imagesAndFonts, limit: 100000 }),
+                    sass({
+                        output: outputFormat === "amd",
```

Comment:
Seems like the initial case is broken now. The output should be only in production builds.
```suggestion
                        output: production && outputFormat === "amd",
```